doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/ref_Storage.adoc

```diff
@@ -0,0 +1,31 @@
+[[Storage]]
```

Comment:
Can this file be deleted now that we're just using links to the RHV content?

Reply:
Yup. done